packages/http-client-csharp/emitter/src/sdk-context-options.ts

```diff
@@ -1,32 +0,0 @@
-import { CreateSdkContextOptions } from "@azure-tools/typespec-client-generator-core";
-
-export let defaultSDKContextOptions: CreateSdkContextOptions = {
-  versioning: {
-    previewStringRegex: /$/,
-  },
-  additionalDecorators: [],
-};
-
-/**
- * Sets the SDK context options to be used by the CSharp emitter.
- * @param options - The SDK context options.
- * @beta
- */
-export function setSDKContextOptions(options: CreateSdkContextOptions) {
-  defaultSDKContextOptions = {
```

Comment:
Should we still do the merge the way that this function did?

Reply:
since there is no default option in MTG emitter, there is no need to merge